00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
I usually prefer to have these envvars end up with _PATH. Or is that not a PATH ? In addition, I started to think that we might want to define a ROCK_PREFIX_PATH variables from which all the other search paths are derived (the way e.g. KDE does it). That would reduce the proliferation of envvars ...